core/kazoo_media/src/kz_media_recording.erl

```diff
@@ -517,18 +517,23 @@ store_url(#state{doc_db=Db
     Options = [{'plan_override', Handler}],
     kz_media_url:store(Db, {<<"call_recording">>, MediaId}, MediaName, Options).
 
--spec should_store_recording() -> store_url().
 -spec should_store_recording(api_binary()) -> store_url().
-should_store_recording(Url) ->
+-spec should_store_recording(api_binary(), api_binary()) -> store_url().
+should_store_recording(AccountId, Url) ->
     case kz_util:is_empty(Url) of
-        'true' -> should_store_recording();
+        'true' -> should_store_recording(AccountId);
         'false' -> {'true', 'other', Url}
     end.
 
-should_store_recording() ->
+should_store_recording('undefined') ->
     case kapps_config:get_is_true(?CONFIG_CAT, <<"store_recordings">>, 'false') of
         'true' -> {'true', 'local'};
         'false' -> 'false'
+    end;
+should_store_recording(AccountId) ->
```

Comment:
@lazedo As per our call I am not a fan of this but I think we have an agreement here :) I look forward to the update!